include/boost/beast/core/impl/flat_buffer.ipp

```diff
@@ -284,7 +321,7 @@ prepare(std::size_t n) ->
 template<class Allocator>
 void
 basic_flat_buffer<Allocator>::
-consume(std::size_t n)
+consume(std::size_t n) noexcept
```

Comment:
Also a questionable that all consumes are noexcept

Reply:
It only performs a possible deallocation. Can deallocate throw?